src/app_service/service/message/dto/pinned_message_update.nim

```diff
@@ -0,0 +1,28 @@
+{.used.}
+
+import json
+
+include ../../../common/json_utils
+
+type PinnedMessageUpdateDto* = object
+  chatId*: string
+  messageId*: string
+  localChatId*: string # not sure what's this and do we need this at all
```

Comment:
This is the chatId for the current user. In practice, it only affects the 1-1 chats, because the chat ID is different for the sender and the receiver, since the chatId depends on who sent the message, but `localChatId` is always the pubkey of the other person (not the connected user)

Reply:
@jrainville just to confirm since I am not sure that I completely understood you, so when we receive list of pinned messages does that mean if `chatType` refers to 1:1 chat we should use `localChatId` (what is `chatId` in that case?) and in all other cases we should use `chatId`?